src/lumigo_opentelemetry/resources/span_processor.py

```diff
@@ -0,0 +1,40 @@
+from opentelemetry.trace import Span
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+from lumigo_opentelemetry import logger
+
+
+# A span attributes that if is set to True, the span will not be exported
+SKIP_EXPORT_SPAN_ATTRIBUTE = "SKIP_EXPORT"
```

Comment:
can we import this string from sdk somehow (instead of declaring it)?

Reply:
This isn't a standard attribute from the SDK, we made up this attribute

Reply:
All the standard library methods of skipping an export don't work for our case, so we are using a custom attribute.
See the discussion on the DR page (Ken responded there)
https://www.notion.so/DR-Drop-OTEL-empty-SQS-polling-spans-7bc799481a024fc1a6056c9dea5e15d7?d=891dfb3b51df428aa8ce83105132867c&pvs=4#52c9f6bc6d344665b7af4a3d43be0be3